apps/site/components/Common/Searchbox/ChatInput/index.tsx

```diff
@@ -0,0 +1,73 @@
+'use client';
+
+import { PaperAirplaneIcon } from '@heroicons/react/20/solid';
+import { PauseCircleIcon } from '@heroicons/react/24/solid';
+import { PromptTextArea, Suggestions } from '@orama/ui/components';
+import { useChat } from '@orama/ui/hooks';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+import { useEffect, useRef } from 'react';
+
+import styles from './index.module.css';
+
+export const ChatInput: FC = () => {
+  const t = useTranslations();
+  const {
+    context: { interactions },
+  } = useChat();
+  const textareaRef = useRef<HTMLTextAreaElement>(null);
+
+  const suggestions = [
+    t('components.search.suggestionOne'),
+    t('components.search.suggestionTwo'),
+    t('components.search.suggestionThree'),
+  ];
+
+  const hasInteractions = !!interactions?.length;
+
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
Please also add a cleanup function of the effect to cancel the setTimeout during component unmount.